client/me/billing-history/utils.jsx

```diff
@@ -50,3 +52,26 @@ export const groupDomainProducts = ( originalItems, translate ) => {
 		} ),
 	];
 };
+
+export function renderTransactionAmount( transaction, { translate, addingTax = false } ) {
+	if ( ! config.isEnabled( 'show-tax' ) || ! transaction.tax_amount ) {
```

Comment:
Note to self: We need to revisit this logic. Particularly with future payments, we may know that we'll need to apply tax, but not the exact amount (because it could change), so we'll want a test for whether or not we should show tax, and then display a generic `"(+tax)"` (without an amount) in this case.